pom.xml

```diff
@@ -303,18 +347,167 @@
+            </properties>
+            <users>
+              <user>
+                <name>Administrator</name>
+                <password>${maven.kie.server.password}</password>
+                <roles>
+                  <role>kie-server</role>
+                  <role>guest</role>
+                  <role>Administrators</role>
+                </roles>
+              </user>
+              <user>
+                <name>${maven.kie.server.username}</name>
+                <password>${maven.kie.server.password}</password>
+                <roles>
+                  <role>kie-server</role>
+                </roles>
+              </user>
+            </users>
+          </configuration>
+        </configuration>
+        <executions>
+          <execution>
+            <id>start-container</id>
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>start</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>stop-container</id>
+            <phase>post-integration-test</phase>
+            <goals>
+              <goal>stop</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>com.bazaarvoice.maven.plugins</groupId>
+        <artifactId>process-exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>start-process-migration-service</id>
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>start</goal>
+            </goals>
+            <configuration>
+              <name>ProcessMigrationService</name>
+              <healthcheckUrl>${maven.pim.server.http.health}</healthcheckUrl>
+              <waitForInterrupt>false</waitForInterrupt>
+              <waitAfterLaunch>300</waitAfterLaunch>
+              <processLogFile>${project.build.directory}/pim.log</processLogFile>
+              <arguments>
+                <argument>java</argument>
+                <argument>-Dkieservers[0].host=${maven.kie.server.http.url}</argument>
+                <argument>-Dkieservers[0].username=${maven.kie.server.username}</argument>
+                <argument>-Dkieservers[0].password=${maven.kie.server.password}</argument>
+                <argument>-Dquarkus.http.port=${maven.pim.server.http.port}</argument>
+                <argument>-jar</argument>
+                <argument>${project.build.directory}/quarkus-app/quarkus-run.jar</argument>
+              </arguments>
+            </configuration>
+          </execution>
+          <execution>
+            <id>stop-process-migration-service</id>
+            <phase>post-integration-test</phase>
+            <goals>
+              <goal>stop-all</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
         <configuration>
+          <skipITs>${maven.skip.it.tests}</skipITs>
```

Comment:
They are already skipped by default. `custom-container` is activated by default which disables both cargo and process-exec-maven-plugin:

https://github.com/kiegroup/process-migration-service/blob/e7b9c95f80fe01d6de744953b87b686b14cde59b/pom.xml#L798

I'm keeping them as enabled to run them in case we want to run IT tests locally by using `wildfly` profile